### DISCUSSION TITLE
Where is .NET Core cross-platform development workload?

### DISCUSSION THREAD

**Discussion by vijaya-lakshmi-venkatraman** [dotnet/msbuild#8207] [Q&A] [answered]
HI,
I am trying to build the repo locally
I see that one of the dependencies mentioned is .NET Core cross-platform development workload
I am unable to locate it in my VS 2022 Professional edition.
Is it renamed to something else?

Thanks
Viji

**Comment by rainersigwald** [comment, marked as answer]
I think either (maybe both?) of `ASP.NET and web development` and `.NET Desktop Development` are the modern equivalent. Can you try that and let us know, and then we can get the doc updated?
